torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -757,7 +757,7 @@ def aot_dispatch_autograd(
 
             if fake_mode is not None:
                 tensorify_python_scalars(fx_g, fake_mode.shape_env, fake_mode)
-            fw_module, bw_module = aot_config.partition_fn(
+            fw_module, bw_module, _ = aot_config.partition_fn(
```

Comment:
chatted offline - we can probably just compute this info outside of the partitioner, in the same way we do it already in AOTAutograd: https://github.com/pytorch/pytorch/blob/main/torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py#L447